src/main/java/edu/cornell/kfs/fp/businessobject/PurchasingDataBase.java

```diff
@@ -0,0 +1,525 @@
+      } else {
+          this.dutyAmount = KualiDecimal.ZERO;
+      }
+    }
+
+
+    /**
+     * @return the destinationPostalZipCode
+     */
+    public String getDestinationPostalZipCode() {
+      return destinationPostalZipCode;
+    }
+
+
+    /**
+     * @param destinationPostalZipCode the destinationPostalZipCode to set
+     */
+    public void setDestinationPostalZipCode(String destinationPostalZipCode) {
+      this.destinationPostalZipCode = destinationPostalZipCode;
+    }
+
+
+    /**
+     * @return the shipFromPostalZipCode
+     */
+    public String getShipFromPostalZipCode() {
+      return shipFromPostalZipCode;
+    }
+
+
+    /**
+     * @param shipFromPostalZipCode the shipFromPostalZipCode to set
+     */
+    public void setShipFromPostalZipCode(String shipFromPostalZipCode) {
+      this.shipFromPostalZipCode = shipFromPostalZipCode;
+    }
+
+
+    /**
+     * @return the destinationCountryCode
+     */
+    public String getDestinationCountryCode() {
+      return destinationCountryCode;
+    }
+
+
+    /**
+     * @param destinationCountryCode the destinationCountryCode to set
+     */
+    public void setDestinationCountryCode(String destinationCountryCode) {
+      this.destinationCountryCode = destinationCountryCode;
+    }
+
+
+    /**
+     * @return the uniqueVATInvoice
+     */
+    public String getUniqueVATInvoice() {
+      return uniqueVATInvoice;
+    }
+
+
+    /**
+     * @param uniqueVATInvoice the uniqueVATInvoice to set
+     */
+    public void setUniqueVATInvoice(String uniqueVATInvoice) {
+      this.uniqueVATInvoice = uniqueVATInvoice;
+    }
+
+
+    /**
+     * @return the orderDate
+     */
+    public Date getOrderDate() {
+      return orderDate;
+    }
+
+
+    /**
+     * @param orderDate the orderDate to set
+     */
+    public void setOrderDate(Date orderDate) {
+      this.orderDate = orderDate;
+    }
+    
+
+    public void setOrderDate(String orderDate) {
+      if (StringUtils.isNotBlank(orderDate)) {
+        SqlDateConverter converter = new SqlDateConverter();
+        converter.setPattern("yyyyMMdd");
+        this.orderDate = (Date) converter.convert(Date.class, orderDate);
+      }
+    }
+
+
+    /**
+     * @return the itemDescriptor
+     */
+    public String getItemDescriptor() {
+      return itemDescriptor;
+    }
+
+
+    /**
+     * @param itemDescriptor the itemDescriptor to set
+     */
+    public void setItemDescriptor(String itemDescriptor) {
+      this.itemDescriptor = itemDescriptor;
+    }
+
+
+    /**
+     * @return the quantity
+     */
+    public Integer getQuantity() {
+      return quantity;
+    }
+
+    /**
+     * @param quantity the quantity to set
+     */
+    public void setQuantity(Integer quantity) {
+      this.quantity = quantity;
+    }
+
+    /**
+     * @param quantity the quantity to set
+     */
+    public void setQuantity(String quantity) {
+      this.quantity = Integer.parseInt(quantity);
```

Comment:
Implicit conversion of primitive int to an Integer object; you can use Integer.valueOf() to get an explicit Integer object instead. Also, this setter is not consistent with some of the others that have special handling for blank values; is this intentional?